process/block/shardblock.go

```diff
@@ -251,6 +251,11 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
+	log.Info(fmt.Sprintf("processed metaBlock %d out of %d", len(processedMetaHdrs), len(header.MetaBlockHashes)))
```

Comment:
Remove these prints which were uses only for quick debugging purpose

Reply:
done